config.yaml.defaults

```diff
@@ -19,6 +19,11 @@ app:
   # this endpoint does not actually do anything -- it is just for testing
   sedm_endpoint: http://pharos.caltech.edu/add_fritz
 
+  lt_username: null
```

Comment:
presumably different allocations will have different usernames and passwords?

Reply:
@dannygoldstein are you thinking of passing the username and passwords through? or different usernames and passwords in secrets? or something else?